muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/profile/body/application/components/work-list-section.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react";
+import { ButtonPill } from "~/components/general/button";
+import Link from "~/components/general/link";
+import moment from "~/lib/moment";
+import { StoredWorklistItem, WorklistBillingState, WorklistSection } from "~/reducers/main-function/profile";
+import WorkListRow from "./work-list-row";
+import SubmitWorklistItemsDialog from "../../../dialogs/submit-worklist-items";
+import { i18nType } from "~/reducers/base/i18n";
+
+function sortBy(data: StoredWorklistItem[], property: string, direction: "asc" | "desc"): StoredWorklistItem[] {
+  let actualProperty = property || "entryDate";
+
+  return [...data].sort(
+    (a: any, b: any) => {
+      if (actualProperty === "entryDate") {
+        // this gives a numeric difference
+        const status = moment(a[actualProperty]).diff(b[actualProperty]);
+        // reverse if the direction is wrong
+        return direction === "asc" ? status : -status;
+      } else {
+        const status = a[actualProperty].localeCompare(b[actualProperty]);
+        // reverse if the direction is wrong
+        return direction === "asc" ? status : -status;
+      }
+    }
+  )
+}
+
+interface IWorkListSectionProps {
```

Comment:
Consider for code conventions if there are. Prefix "I" is unnecessary and differ from other components. And its not even recommended anymore. So WorkListSectionProps should be fine. And check and remove all prefix from whole pr

Reply:
Done, don't know why "I" prefix were used but it's removed from every profile view's component (8 files affected in total).